location/src/androidLibMain/kotlin/location/GoogleLocationProvider.kt

```diff
@@ -36,48 +36,41 @@ import kotlinx.coroutines.flow.flatMapLatest
 import kotlinx.coroutines.flow.flowOf
 import kotlin.time.Duration
 import kotlin.time.Duration.Companion.milliseconds
-import kotlin.time.Duration.Companion.seconds
 
 /**
  * A [LocationProvider] using the Google Location Services
  * @param context The [Context] to run this [LocationProvider] in.
  * @param settings The [GoogleLocationProvider.Settings] to use to configure this location provider.
  */
-class GoogleLocationProvider(private val context: Context, private val settings: Settings = Settings.default) : LocationProvider {
+class GoogleLocationProvider(private val context: Context, private val settings: Settings) : LocationProvider {
 
     /**
      * Settings for a [GoogleLocationProvider]
-     * @param interval The desired interval of location updates.
```

Comment:
Max update and min update delay are set to interval value by default. Since we now want to actually set an interval I came to the conclusion this is the most reliable way to set it closer to what you want.